### PR TITLE
OCPBUGS#1418: Updated KCS link for SSH

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.8+

Issue: https://issues.redhat.com/browse/OCPBUGS-1418

Link to docs preview: https://50710--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html#using-machineconfigs-to-change-machines

### CODE REVIEW

**Comment by snarayan-redhat** [comment]
Closing and reopening PR since Travis isn't running checks

**Comment by sergiordlr** [comment]
looks good to me

**Review by JoeAldinger — COMMENTED** [review]
/label peer-review-done
/lgtm

File: post_installation_configuration/machine-configuration-tasks.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ include::modules/checking-mco-status.adoc[leveloffset=+2]
...
 
-You can use the tasks in this section to create `MachineConfig` objects that modify files, systemd unit files, and other operating system features running on {product-title} nodes. For more ideas on working with machine configs, see content related to link:https://access.redhat.com/solutions/4510281[updating] SSH authorized keys, link:https://access.redhat.com/verify-images-ocp4[verifying image signatures], link:https://access.redhat.com/solutions/4727321[enabling SCTP], and link:https://access.redhat.com/solutions/5170251[configuring iSCSI initiatornames] for {product-title}.
+You can use the tasks in this section to create `MachineConfig` objects that modify files, systemd unit files, and other operating system features running on {product-title} nodes. For more ideas on working with machine configs, see content related to link:https://access.redhat.com/solutions/3868301[updating] SSH authorized keys, link:https://access.redhat.com/verify-images-ocp4[verifying image signatures], link:https://access.redhat.com/solutions/4727321[enabling SCTP], and link:https://access.redhat.com/solutions/5170251[configuring iSCSI initiatornames] for {product-title}.
```
Comment (JoeAldinger) [inline post_installation_configuration/machine-configuration-tasks.adoc:26]:
```suggestion
You can use the tasks in this section to create `MachineConfig` objects that modify files, systemd unit files, and other operating system features running on {product-title} nodes. For more ideas on working with MachineConfigs, see content related to link:https://access.redhat.com/solutions/3868301[updating] SSH authorized keys, link:https://access.redhat.com/verify-images-ocp4[verifying image signatures], link:https://access.redhat.com/solutions/4727321[enabling SCTP], and link:https://access.redhat.com/solutions/5170251[configuring iSCSI initiatornames] for {product-title}.
```

Comment (snarayan-redhat) [reply post_installation_configuration/machine-configuration-tasks.adoc:26]: Thanks Joe! `machine configs` seems to be a valid common noun which is used across the repo.

Comment (maxwelldb) [reply post_installation_configuration/machine-configuration-tasks.adoc:26]: Correct: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Note that "MachineConfigs," if it were to be used here, would have to be "`MachineConfig` resources" or something similar.

**Review by maxwelldb — COMMENTED** [review]
If you wanted to clean up the link language in on L26, I wouldn't complain, but that's outside of the scope of this PR. 

/lgtm

File: post_installation_configuration/machine-configuration-tasks.adoc (hunk 1 above)
Comment (JoeAldinger) [reply post_installation_configuration/machine-configuration-tasks.adoc:26]: Sorry, @snarayan-redhat . @maxwelldb thanks for the reference.

**Pull request merged**